Tests/Resources/Fixtures/config/full.yml

```diff
@@ -0,0 +1,9 @@
+http_client:
+    main_alias:
```

Comment:
seems i did not copy that one from the default repo. will add it.

however, we have a bit of an issue here as symfony CS says 4 spaces. so
for a symfony bundle i would tend to follow symfony CS in that regard...